src/filter/ticket.rs

```diff
@@ -76,6 +77,18 @@ impl FetchPlaceholders {
         }
     }
 
+    /// Returns the list of placeholders.
+    pub fn pass_with_placeholder<'a>(
+        &self,
+        placeholders: impl Into<Option<&'a [PlaceholderFile<'a>]>>,
+        total: u64,
+    ) -> core::Result<Vec<core::Result<Usn>>> {
+        command::CreatePlaceholders {
+            placeholders: placeholders.into(),
+            total,
+        }
+        .execute(self.connection_key, self.transfer_key)
+    }
```

Comment:
Tickets are used to confirm or deny Windows prompts. In the case of `fetch_data`, I believe there is no prompt to confirm, thus the reason I didn't have a pass function (correct me if I'm wrong). In this scenario, I believe the user should construct the placeholders directly using the dedicated structs.

Reply:
`fetch_data` seems need to comfirm by `CfExecute`, otherwise `fetch_data` will be keep invoked.